python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -220,6 +250,8 @@ def run_failure_sensor(
         job_selection (Optional[List[Union[JobDefinition, GraphDefinition]]]): (deprecated in favor of monitored_jobs)
             The jobs that will be monitored by this failure sensor. Defaults to None, which means
             the alert will be sent when any job in the repository fails.
+        monitored (Optional[List[Union[RepositorySelector, JobSelector]]]): Other repositories or jobs in other repositories that this
```

Comment:
todo - update doc string for `monitored_jobs` and `job_selection` to say that it's for jobs in the same repo as the sensor